packages/calling-stateful-client/review/stable/calling-stateful-client.api.md

```diff
@@ -106,6 +107,9 @@ export type CreateViewResult = {
     view: VideoStreamRendererView;
 };
 
+// @public
+export type DeclarativeCallAgent = CallAgent & {};
```

Comment:
Do you _have_ to add this to stable API right now?

You should be able to add the function overload to @beta only instead, right?

This makes for an awkward stable API.

Reply:
Made it cleaner as suggested